src/components/AddBook/addbook.js

```diff
@@ -23,24 +23,28 @@ const AddBook = () => {
 
   return (
     <form onSubmit={submitHandler}>
-      <p>ADD NEW BOOK</p>
-      <input
-        type="text"
-        name="title"
-        value={title}
-        onChange={(e) => setTitle(e.target.value)}
-        placeholder="Book title"
-      />
-      <input
-        type="text"
-        name="author"
-        value={author}
-        onChange={(e) => setAuthor(e.target.value)}
-        placeholder="Book author"
-      />
-      <button type="submit" className="btn-submit">
-        Add Book
-      </button>
+      <p className="form-title">ADD NEW BOOK</p>
+      <div className="form">
+        <input
+          type="text"
+          name="title"
+          className="title"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+          placeholder="Book title"
+        />
+        <input
+          type="text"
+          name="author"
+          className="author"
+          value={author}
+          onChange={(e) => setAuthor(e.target.value)}
+          placeholder="Book author"
+        />
+        <button type="submit" className="btn-submit" id="btn-add">
+          <p className="btn-info">ADD BOOK</p>
+        </button>
```

Comment:
- [ ] Good job implementing the changes requested however, as mentioned by the previous reviewer please make sure the form is properly aligned currently the left side is properly aligned but on the right side of the form the Add Book button is not properly aligned with the books container please see :point_down:  
![image](https://user-images.githubusercontent.com/97521557/230622044-aaec9bd3-f249-466e-8cb6-44c0fe0b4556.png)



Zeplin design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230621270-c33a4c94-302d-4727-89a1-57ad314318ad.png)

Reply:
Hello @PinkMoon25 👋, [this here](https://drive.google.com/file/d/1nCzUXXGD_eZ5-ma3l-0S3vQpj2qX3dFw/view?usp=sharing) is how the project appears from on my PC after making the changes requested by @HAMMAS-SALEEM , I have even tried again to implement  the changes you requested and that's how it still appears from my side, kindly advice🙂